docker/package/packages.py

```diff
@@ -82,6 +82,20 @@
     node_units.append(SystemdUnit(suffix=network,
                                   service_file=service_file,
                                   startup_script="tezos-node-start"))
+
+node_units.append(SystemdUnit(suffix="custom",
+                              service_file=ServiceFile(
+                                  Unit(after=["network.target"], requires=[],
+                                       description=f"Tezos node with custom config"),
+                                  Service(environment=[
+                                      "DATA_DIR=/var/lib/tezos/node-custom",
+                                      "NODE_RPC_ADDR=127.0.0.1:8732",
+                                      "CERT_PATH=", "KEY_PATH=",
+                                      "CUSTOM_NODE_CONFIG="],
+                                          exec_start="/usr/bin/tezos-node-start",
+                                          state_directory="tezos", user="tezos")),
+                              startup_script="tezos-node-start"))
```

Comment:
From what I can tell there's very little difference between this and any other `network` node unit.

I suppose adding `"custom"` to the `networks` list would be a bit improper (and require an ugly check in the loop), but couldn't the common parts be moved to a function instead?

Reply:
Moved common parts to a `mk_node_unit` function